pkg/db/models/prow.go

```diff
@@ -82,12 +83,12 @@ type ProwJobRunTest struct {
 
 	// ProwJobRunTestOutput collect the output of a failed test run. This is stored as a separate object in the DB, so
 	// we can keep the test result for a longer period of time than we keep the full failure output.
-	ProwJobRunTestOutput *ProwJobRunTestOutput
+	ProwJobRunTestOutput *ProwJobRunTestOutput `gorm:"constraint:OnDelete:CASCADE;"`
 }
 
 type ProwJobRunTestOutput struct {
 	gorm.Model
-	ProwJobRunTestID uint `gorm:"constraint:OnDelete:SET NULL;"`
+	ProwJobRunTestID uint `gorm:"index"`
```

Comment:
Is it intentional to remove the index here?

Reply:
I'm removing the constraint (it's in the wrong place) and adding an index.